frontend/src/Event/MeetEventDisplay.jsx

```diff
@@ -210,6 +226,8 @@ const MeetEventDisplay = () => {
             eventName={eventData[selectedEventIndex].name}
             eventId={eventData[selectedEventIndex].id}
             onBack={handleBackToEvents}
+            setGenerateHeatTrigger={setGenerateHeatTrigger}
+            switchViews={handleSwitchGenerateToDetails}
```

Comment:
Similarly here, I think this prop would be better defined  with a more general name like handleGenerateHeatProcessCompletion, something like that, what do you think?

Reply:
To handle the completition of the process we need to use setGenerateHeatsTrigger and switchViews. 
But now that you mention this, I was thinking that setGenerateHeatsTrigger does not need to be pass as a prop, it could be trigger inside handleSwitchGenerateToDetails. I wil give it a try and then change the name of that prop.